### PR TITLE
Feature/csv export

### PR DESCRIPTION
**Pull request by jonmitten**
This PR supplies a function to export the library to a csv file. 

The csv file is hard-coded `library.csv`, but can be updated to be parameterized in a future implementation.

### CODE REVIEW

File: create_catalog.py [hunk 1]
```diff
@@ -19,8 +19,15 @@
...
-
+    '''
+    Process special characters used on the HTML encoding to
+    ASCII or UTF-8
+    '''
+    try:
+        return unicodedata.normalize('NFKD', unicode(text)).encode('ascii', 'ignore')
+    except NameError as ne:
+        print(ne)
+        return str(text)
```
Comment (jonmitten) [inline create_catalog.py:27]: This satisfies the Python 3 unicode requirement.

Comment (jedmitten) [reply create_catalog.py:27]: Nice!

Comment (jonmitten) [reply create_catalog.py:27]: Actually, this is outdated... 
I've got a much shorter fix for this. Not sure why it hasn't made it into the PR...

**Comment by jedmitten** [comment]
Rejected per @jonmitten request to make different changes

**Pull request closed without merging**